tools/c7n_gcp/c7n_gcp/resources/gke.py

```diff
@@ -72,9 +72,23 @@ def refresh(cls, client, resource):
     def augment(self, resources):
         if not resources:
             return []
+        session = local_session(self.session_factory)
+        project = session.get_default_project()
+        location = resources[0]["location"]
+        server_config_client = session.client(
+            self.resource_type.service, self.resource_type.version, "projects.locations"
+        )
+        response = server_config_client.execute_command(
```

Comment:
so rather than do this in augment, it would be better to have this as a dedicated filter (`server-config`). with an augment we have to issue the api calls even if the user didn't care about this data.

Reply:
Updated to use a dedicated filter called server-config